nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ndarray/BaseSparseNDArray.java

```diff
@@ -1673,12 +1679,12 @@ public INDArray getRow(long i) {
 
     @Override
     public int columns() {
-        return columns;
+        return (int) columns;
```

Comment:
We can switch those to long as well since we're not worried about backward compatibility.

Reply:
That implements a method of the `INDArray` interface. Changing the signature has far reaching effects I believe.

Reply:
Oops, you're right.